src/_includes/layouts/base.html

```diff
@@ -77,6 +77,8 @@
     <script>
             (function () {
                 var theme = window.localStorage.getItem("theme");
+                var isSystemTheme = window.localStorage.getItem("system-theme") === "true";
```

Comment:
```suggestion
                var isSystemTheme = window.localStorage.getItem("system-theme") === "true";
```
We don't need another key system-theme instead we can use the existing theme key and set the value as "system"